webapi/src/main/scala/org/knora/webapi/slice/admin/api/ListsEndpointsHandlers.scala

```diff
@@ -5,17 +5,88 @@
 
 package org.knora.webapi.slice.admin.api
 
+import zio.Random
+import zio.Task
+import zio.ZIO
 import zio.ZLayer
 
+import dsp.errors.BadRequestException
+import org.knora.webapi.messages.admin.responder.listsmessages.CanDeleteListResponseADM
+import org.knora.webapi.messages.admin.responder.listsmessages.ListItemDeleteResponseADM
+import org.knora.webapi.messages.admin.responder.listsmessages.ListNodeCommentsDeleteResponseADM
+import org.knora.webapi.messages.admin.responder.listsmessages.NodeInfoGetResponseADM
+import org.knora.webapi.messages.admin.responder.listsmessages.NodePositionChangeResponseADM
 import org.knora.webapi.responders.admin.ListsResponder
+import org.knora.webapi.slice.admin.api.Requests.*
 import org.knora.webapi.slice.admin.domain.model.KnoraProject.ProjectIri
+import org.knora.webapi.slice.admin.domain.model.ListProperties.ListIri
+import org.knora.webapi.slice.admin.domain.model.User
+import org.knora.webapi.slice.admin.domain.service.KnoraProjectRepo
+import org.knora.webapi.slice.common.api.AuthorizationRestService
 import org.knora.webapi.slice.common.api.HandlerMapper
 import org.knora.webapi.slice.common.api.PublicEndpointHandler
-import org.knora.webapi.slice.search.api.SearchEndpointsInputs.InputIri
+import org.knora.webapi.slice.common.api.SecuredEndpointHandler
+
+final case class ListRestService(
+  auth: AuthorizationRestService,
+  listsResponder: ListsResponder,
+  projectRepo: KnoraProjectRepo
+) {
+  def listChange(iri: ListIri, request: ListChangeRequest, user: User): Task[NodeInfoGetResponseADM] = for {
+    _ <- ZIO.fail(BadRequestException("List IRI in path and body must match")).when(iri != request.listIri)
+    project <- projectRepo
+                 .findById(request.projectIri)
+                 .someOrFail(BadRequestException("Project not found"))
```

Comment:
```suggestion
    _ <- ZIO.fail(BadRequestException("List IRI in path and body must be the same.")).when(iri != request.listIri)
    project <- projectRepo
                 .findById(request.projectIri)
                 .someOrFail(BadRequestException("Project not found."))
```
IMO it sounds more explicit.

Reply:
Thanks for the suggestion.

I will not apply it because this would change the existing response and I am not sure if there is a test which would break.